exp_hub.html

```diff
@@ -58,7 +58,8 @@
 
 		// start the experiment
 		jsPsych.init({
-		  experiment_structure: experiments,
+		  //experiment_structure: experiments,
+		  timeline: experiments,
```

Comment:
The timeline is the replacement for another variabel "experiment" or
something like that. In this case it is just a variable rename.

On Tue, Jan 12, 2016 at 1:25 PM, Vanessa Sochat <notifications@github.com>
wrote:

> In exp_hub.html
> <https://github.com/expfactory/expfactory-battery/pull/11#discussion_r49515945>
> :
>
> > @@ -58,7 +58,8 @@
> >
> >  		// start the experiment
> >  		jsPsych.init({
> > -		  experiment_structure: experiments,
> > +		  //experiment_structure: experiments,
> > +		  timeline: experiments,
>
> We didn't have timeline before - what exactly is this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/expfactory/expfactory-battery/pull/11/files#r49515945>
> .
>



-- 
Best,
Ian